website/docs/sat/constellations.md

```diff
@@ -0,0 +1,3 @@
+# Constellations
+
+Sat is designed to run in a constellation, i.e. a meshed cluster of instances. This enables very interesting network topologies which can run applications in massively distributed and 'edge' environments. This repo includes the `constd` tool, which is an experiment-atop-experiment constellation manager that can run [Atmo](https://github.com/suborbital/atmo) applications in a distributed manner using a Sat constellation. You can learn more [here](./constd).
```

Comment:
The link at the end of this should actually link to the `constd` folder of the `sat` repo, as it's not a file in this project